classes/controllers/grid/issues/IssueGridHandler.inc.php

```diff
@@ -280,6 +281,41 @@ function deleteIssue($args, $request) {
 	}
 
 	/**
+	 * An action to edit a issue's pub ids
+	 * @param $args array
+	 * @param $request PKPRequest
+	 * @return JSONMessage JSON object
+	 */
+	function pubIds($args, $request) {
```

Comment:
Mind changing this to "identifiers" rather than "pubIds"? (And the "update" function below.) We'll also be moving the public IDs into this, so it's more general than just the pub ID plugins.